notebook/templates/edit.html

```diff
@@ -8,7 +8,7 @@
   <link rel="icon" type="image/x-icon" href="{{ base_url | escape }}static/favicons/favicon-file.ico" class="favicon">
   {% endblock %}
 </head>
-<body>
+<body data-edit="edit">
```

Comment:
My reasoning for doing it this way is that the [notebook template](https://github.com/jupyter/notebook/blob/main/packages/notebook-extension/style/base.css#L26) has an attribute just like this. It is only ever referenced in [this](https://github.com/jupyter/notebook/blob/main/packages/notebook-extension/style/base.css#L26) and [this](https://github.com/jupyter/notebook/blob/main/packages/application/style/base.css#L73) CSS file for the purposes of styling.